openwisp_monitoring/monitoring/tests/test_models.py

```diff
@@ -77,11 +91,18 @@ def test_write(self):
         self.assertEqual(measurement['value'], 2)
 
     def test_general_write(self):
-        m = self._create_general_metric(name='Sync test')
+        # TODO: Remove or modify this test
```

Comment:
Why?

Reply:
It needs to modified since the `query` used in the test is specific for `influxDB`. I don't recall though why I mentioned `remove`